### PR TITLE
Initial implementation

### PR DESCRIPTION
**Pull request by vigan-abd**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,47 @@
+{
+  "name": "@bitfinex/lib-util-err-js",
+  "version": "0.0.1",
+  "description": "Bitfinex lib utility for handling error messages",
+  "main": "index.js",
+  "scripts": {
+    "docs": "jsdoc2md --no-cache --files ./src/*.js ./src/**/*.js > docs/DEFINITIONS.md",
```
Comment (robertkowalski) [inline package.json]: maybe its better to not check those generated files in? they clutter PRs and usually docs are generated locally and the compiled result is not checked into source control

Comment (robertkowalski) [reply package.json]: cris started with checking in those docs in in the HF and its a nightmare, especially when the docs are large and someone runs it and the `generated at Friday` comment is everything that has changed

Comment (vigan-abd) [reply package.json]: Sure I'll add docs dir to gitignore then

**Review by ezewer — APPROVED** [review]
Looks good to me!

**Pull request merged**